pkg/cmd/shell.go

```diff
@@ -59,8 +60,28 @@ func NewShellCmd(reader TargetReader, ioStreams IOStreams) *cobra.Command {
 			if client, err = target.K8SClient(); err != nil {
 				return err
 			}
+
+			gardenClientset, err := target.GardenerClient()
+			checkError(err)
+			var shoot *gardencorev1alpha1.Shoot
```

Comment:
This code fragment below is duplicated across the project several times - for example in ssh.go. Can we extract the code fragment to prevent duplication?